src/main/java/backend/Dispatcher.java

```diff
@@ -0,0 +1,111 @@
+package backend;
```

Comment:
I think backend is appropriate given the other stuff it holds, but it also holds a ton of stuff. I could subdivide the packages, but I think that's a refactor for a future date-- like when our MVP is working and we're polishing this sucker up.

Keeping this in mind.